securedrop/upload-screenshots.py

```diff
@@ -23,7 +29,7 @@
 # filename into the canonical title we give that screenshot in Weblate.
 #
 # Example conversion: "source-session_timeout.png" -> "source: session timeout"
-CANONICALIZATION_RULES = ((r"\.png$", ""), (r"-", ": "), (r"_", " "))
+CANONICALIZATION_RULES = [(r"\.png$", ""), (r"-", ": "), (r"_", " ")]
```

Comment:
A tuple with an arbitrary length can't be typed so I switched to a list.

Reply:
I think you could also have used `Tuple[Tuple[str, str], ...]` for this, but this is fine.